R/utilities-simulation.R

```diff
@@ -425,6 +425,22 @@ getAllParametersForSensitivityAnalysisMatching <- function(paths, simulation) {
   )
 }
 
+#' Get the paths of all state variable quantities of the simulation
+#'
+#' @param simulation \code{Simulation} object
+#' @details List of paths of all molecules in all compartments and all parameters that are
+#' state variables.
+#'
+#' @return A list of paths
+#' @export
+getAllStateVariablesPaths <- function(simulation) {
+  validateIsOfType(simulation, type = "Simulation")

```

Comment:
you can pass a ref to the R6 class directly here (e..g validateIsOfType(simulation, Simulation) which removes the use of "magic" string

Reply:
Done